tapioca_{{cookiecutter.service_name|lower|replace(' ', '_')}}/setup.py

```diff
@@ -57,9 +57,9 @@ def get_email(package):
 if sys.argv[-1] == 'publish':
     os.system("python setup.py sdist upload")
     args = {'version': get_version(package)}
-    print "You probably want to also tag the version now:"
-    print "  git tag -a %(version)s -m 'version %(version)s'" % args
-    print "  git push --tags"
+    print("You probably want to also tag the version now:")
```

Comment:
Just to be safe, it think it's good to add `from __future__ import print_function` to the top of file